### ISSUE TITLE
[Due 9/20] Update /firefox/mobile/ page w. new strings

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#6083]
[Bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1483945)
Update https://www.mozilla.org/en-US/firefox/mobile/#focus with new strings

Success Criteria:
- [ ] strings are in place and live on 9/20 to correspond to product release
- [ ] localization is required

=======
Remove:
Lightweight
iOS and Android

Save your mobile memory for the memories. Firefox Focus takes up less selfie space on your phone. 

Replace with:
On your side
iOS and Android
Focus is next-level privacy that’s free, always on and always on your side — because it’s backed by Mozilla, the non-profit that fights for your rights on the Web.

**Comment by ejregithub** [comment]
@thejeffreytaylor - can you give the new copy a review, please?

**Comment by thejeffreytaylor** [comment]
Should I review just what's above?

**Comment by lwa-moz** [comment]
@thejeffreytaylor yes, the "Replace with" text above could use your input for clarity/brevity.

**Comment by thejeffreytaylor** [comment]
Here's the update (case included): 

On Your Side
iOS and Android

Focus has next-level privacy that's free, always on, and always on your side. It's a purpose-driven product by Mozilla that's built for you – not your data.
